common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java

```diff
@@ -20,14 +20,19 @@
 
 public class AspectAnnotator implements GameItemAnnotator {
     private static final Pattern ASPECT_PATTERN = Pattern.compile("^§(.)Aspect of .*$");
+    private static final Pattern MYTHIC_ASPECT_PATTERN = Pattern.compile("^§(.)(.*) Embodiment of .*$");
```

Comment:
```suggestion
    private static final Pattern ASPECT_PATTERN = Pattern.compile("^§(.)(?:Aspect|Embodiment) of .*$");
```

What do you think? Or is there another reason?

Reply:
Changing to `§(.*)` works. Thanks

Reply:
Wait a bit. Are you expecting multiple color codes there? Like mythic color + bold? Perhaps `(§.)+` is wiser then.

Reply:
The mythic ones have an archetype name before the "Embodiment" like "Paladin's Embodiment of..."

Reply:
Ah. `§. (?:(?:(.+) Embodiment)|(?:Aspect)) of ...` perhaps?

Reply:
`^§(.)(?:(?:(.+) Embodiment)|(?:Aspect)) of .*$` works?

Reply:
If it works, it works :)